compendium/modules/w04-objects-exercise.tex

```diff
@@ -485,17 +471,15 @@
 \end{minipage}
 
 \begin{REPL}
-> scala -cp introprog_2.13-1.1.4.jar
+> scala -cp introprog_2.13-1.1.4.jar -repl
 scala> val w = new introprog.PixelWindow(400,300,"Tre nyanser av brunt")
 scala> type Pt = (Int, Int)
 scala> var color = java.awt.Color.red
 scala> def rak(p: Pt)(d: Int) = w.line(p._1, p._2, ???, ???, color)
-scala> def fyll(p: Pt)(s: Int) = for (i <- ???){ rak((p._1, ???))(s) }
+scala> def fyll(p: Pt)(s: Int) = for i <- ??? do rak((p._1, ???))(s)
 
-scala> :paste
-object Color {
- ???
-}
+scala> object Color:
+     |   ???
```

Comment:
I have removed all `|` in prints as they dont add anything and just make copy paste harder